docs/site/content/posts/05-tce-10-available.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Tanzu Community Edition 0.10 is Now Available"
+slug: tanzu-community-edition-0.10-available
+date: 2022-02-17
+author: Roger Klorese
+image: /img/laptop-user.jpg
+excerpt: "The Tanzu Community Edition team is pleased to announce the availability of Tanzu Community Edition 0.10. The new version
+enhances usability on your local system, makes it easier to customize
+your platform, and simplifies IP address management across all nodes in
+a cluster."
+tags: ['Roger Klorese']
+---
+
+The Tanzu Community Edition team is pleased to announce the availability
+of [Tanzu Community Edition 0.10](https://tanzucommunityedition.io/download/). The new version
+enhances usability on your local system, makes it easier to customize
+your platform, and simplifies IP address management across all nodes in
+a cluster. Today's release follows the recent [addition of build automation](https://tanzucommunityedition.io/posts/tanzu-community-edition-adds-kpack/)
+capabilities into Tanzu Community Edition as the team continues to
+deliver the power of Tanzu to you in a free, open source distribution.
+
+Here is a quick summary of the highlights of the new release:
+
+- The 0.10 release introduces a new 'unmanaged' cluster type that
+    significantly improves the efficiency of Tanzu Community Edition
```

Comment:
Bullet item continuation lines are over indented (should align with the "T" of "The" in the first line. That said, it appears to render fine, so as long as nothing changes in the renderer, it should be fine.

Reply:
Right. There was a listing issue where there needed to be only 1 space after a bullet and the bullets all started out a that spacing. I saw it was rendering properly so I didn't touch it. I'm happy to make that correction though.

Reply:
looks like all linting has passed resolving